### ISSUE TITLE
remove/update cli command batch-send-payments

### ISSUE THREAD

**Issue by deepthiskumar** [CodaProtocol/coda#4597]
With #4553, the internal flow of local transactions changed causing `batch-send-payments` to send keypair over rpc. We either change it such that it uses graphql or delete if no longer in use. Once this is done, we can remove the serialized versions in user_command_input.ml

**Comment by aneesharaines** [comment]
Can we just remove this for now?

**Comment by aneesharaines** [comment]
Create a separate ticket for the follwup work to reenable properly
